src/api/Gemfile

```diff
@@ -21,6 +21,7 @@ gem 'thinking-sphinx', '> 3.1'
 gem 'kaminari'
 # for abstract HTML
 gem 'haml'
+gem 'haml-rails'
```

Comment:
This shouldn't be under development group?

Reply:
I would say that some things are also for development :thinking: